### ISSUE TITLE
skipLike should be optional in strnumOptions type

### ISSUE THREAD

**Issue by strmer15** [NaturalIntelligence/fast-xml-parser#368]
* [x] Are you running the latest version?
* [x] Have you included sample input, output, error, and expected output? 
* [x] Have you checked if you are using correct configuration?
* [ ] Did you try online [tool](https://naturalintelligence.github.io/fast-xml-parser/)?

### Description
https://github.com/NaturalIntelligence/fast-xml-parser/blob/10bbdeeb6b8ab48eaefc89d63f13724702d52fb1/src/parser.d.ts#L23

The `skipLike` option is required in Typescript, but the `strnum` library makes it optional:

https://github.com/NaturalIntelligence/strnum/blob/a0bf6e5f9843c3aeca5af200114f90d4e819d7df/strnum.js#L10
https://github.com/NaturalIntelligence/strnum/blob/a0bf6e5f9843c3aeca5af200114f90d4e819d7df/strnum.js#L26

The Typescript type should probably be written as
```
type strnumOptions = {
  hex: boolean;
  leadingZeros: boolean,
  skipLike?: RegExp
}
```

### Input
Attempt to use `numParseOptions` without specifying `skipLike`, since it defaults to `undefined` in the JS source.

### Code
```js
import * as xmlParser from 'fast-xml-parser'

xmlParser.parse('<tag>JVBERi0xLjMNCiXi48</tag>', {
  numParseOptions: { hex: false, leadingZeros: true },
})

```

### Output
`Property 'skipLike' is missing in type '{ hex: false; leadingZeros: true; }' but required in type 'strnumOptions'.ts(2741)`

### expected data
No error is expected.

**Would you like to work on this issue?**

- [x] Yes
- [ ] No

[Bookmark](https://github.com/NaturalIntelligence/fast-xml-parser/stargazers) this repository for further updates.
